libzhlgen/test/functions/win32/1.6.9/freetype.zhl

```diff
@@ -15,6 +15,17 @@ static cleanup uint64_t freetype::easy_printCenter_DO_NOT_USE_DIRECTLY(int fontS
 ".8d4c240483e4f0ff71fc5589e55756535181ec????????8b5110":
 static cleanup __cdecl void freetype::easy_printAutoShrink(int fontId, float x, float y, int maxWidth, bool centered, const std::string& text);
 
+"55660fefc9660fefc089e55389cb83e4f083ec20":
```

Comment:
`init` comes before `clean` and that comes long before `easy_measurePrintLines` in Linux, as such, this signature needs to be at the top of the file and **`easy_measurePrintLines` needs to have a `.` added to its signature for optimization so it doesn't restart the scan.**
The order on Linux needs to be:
1. init
2. clean
3. easy_measurePrintLines & everything else
4. font_height
5. font_baseline

Linux:
`freetype::font_data::init`
64: `89c9534889fbf3480f2ac14889f731c989d631d2`
32: `5383ec288b44243c8b5c24300fb7d0c1e810`